samples/cookie-app/js/public/js/cookies.js

```diff
@@ -0,0 +1,84 @@
+function getCookies() {
+  document.getElementById("cookies").innerHTML = document.cookie;
+  if (!cookieStore) {
+    document.getElementById("cookieStore").innerHTML =
+      "Cookie Store API not supported";
+    return;
+  }
+
+  cookieStore.getAll().then((cookies) => {
+    if (cookies.length === 0) {
+      document.getElementById("cookieStore").innerHTML = "No cookies";
+      return;
+    }
+    document.getElementById("cookieStore").innerHTML = JSON.stringify(
+      cookies,
+      null,
+      2
+    );
+  });
+}
+
+function popOutApp() {
+  window.open("https://teams-cookie-app.azurewebsites.net/partitioned-cookies.html");
```

Comment:
This should be point local (Relative path)

Reply:
fixed